org.eclipse.wb.core.java/src/org/eclipse/wb/internal/core/nls/bundle/AbstractBundleSource.java

```diff
@@ -661,6 +677,11 @@ private void saveBundle(BundleInfo bundleInfo) throws Exception {
    * Return list of IResource's that represent bundle files (*.properties) for given bundle name.
    */
   protected final IFile[] getBundleFiles() throws Exception {
+    String source = this.getClass().getName();

```

Comment:
This looks strange: Why not:  "if (getClass().equals(ActivatorSource.class)).."

Reply:
ok, this will work